tools/get.py

```diff
@@ -71,13 +71,13 @@ def get_tool(tool):
     local_path = dist_dir + archive_name
     url = tool['url']
     #real_hash = tool['checksum'].split(':')[1]
-    if 'CYGWIN_NT' in sys_name:
+    if 'CYGWIN_NT' not in sys_name:
```

Comment:
Also as a datapoint while trying to determine how to get this script working for everyone. The script is broken on a clean install of "babun" (cygwin distro) on windows 10 but works if you follow the "not cygwin" path, i.e. with these changes.